tests/e2e/default/translations/enabled-languages.wdio-spec.js

```diff
@@ -0,0 +1,110 @@
+  before(async () => {
+    const settings = {
+      languages: [
+        {
+          locale: 'en',
+          enabled: true,
+        },
+        {
+          locale: 'es',
+          enabled: true,
+        },
+        {
+          locale: 'fr',
+          enabled: true,
+        },
+      ],
+    };
+    await utils.updateSettings(settings, true);
+    await browser.waitUntil(async () => {
+      const { languages } = await utils.getSettings();
+      return isEqual(languages, settings.languages);
+    });
+    await browser.refresh();
+  });
+
+  after(async () => {
+    await utils.revertSettings(true);
+  });
+
+  const toggleLanguage = async (locale, shouldEnable) => {
+    await $('.tab-content > #language-accordion > .panel').waitForDisplayed();
+    const languageHeader = await $(`#locale-${locale}.panel-heading a[data-target="#locale-${locale}-body"]`);
+    await languageHeader.click();
+    const languageAccordion = await $(`#locale-${locale}-body`);
+    await languageAccordion.waitForDisplayed();
+    const buttonLabel = shouldEnable ? 'Enable' : 'Disable';
+    await (await languageAccordion.$(`span=${buttonLabel}`)).click();
+    await browser.waitUntil(async () => {
+      const settings = await utils.getSettings();
+      const language = settings.languages.find(language => language.locale === locale);
+      return language.enabled === shouldEnable;
+    });
+  };
+
+  const disableLanguage = (locale) => toggleLanguage(locale, false);
+  const enableLanguage = (locale) => toggleLanguage(locale, true);
+
+  it('should disable a language and enable another', async () => {
+    // assert English, Spanish, and French are available on the login page
+    let locales = await loginPage.getAllLocales();
+    expect(locales).to.deep.equal([
+      { code: 'en', name: 'English' },
+      { code: 'es', name: 'Español (Spanish)' },
+      { code: 'fr', name: 'Français (French)' },
+    ]);
+    await loginPage.cookieLogin();
+
+    // open admin app on the languages configuration tab
+    await browser.url('/admin/#/display/languages');
+
+    // disable Spanish
+    await disableLanguage('es');
+
+    // enable Swahili
+    await enableLanguage('sw');
+
+    // assert:
+    //   - Spanish has been disabled in the app_settings
+    //   - Swahili has been enabled in the app_settings
+    const settings = await utils.getSettings();
+    expect(settings.languages).to.deep.equal([
+      {
+        locale: 'en',
+        enabled: true,
+      },
+      {
+        locale: 'es',
+        enabled: false,
+      },
+      {
+        locale: 'fr',
+        enabled: true,
+      },
+      {
+        locale: 'sw',
+        enabled: true,
+      },
+    ]);
+
+    // assert:
+    //   - Spanish is not available on the login page
+    //   - Swahili is available on the login page
+    const logoutButton = await $('span=Log out');
+    await logoutButton.click();
+    await loginPage.loginButton().waitForDisplayed();
```

Comment:
I thought so too but we can't because it's meant to log the user out from the webapp, not the admin app

Reply:
Ah, yes!  :facepalm: Not a big deal, but this bit of functionality could also go in `admin.wdio.page.js` since it is specific to the admin console....

Reply:
Ah, nice catch!